var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -79,6 +75,37 @@ class QuantumEspresso(CMakePackage):
         conflicts("^openblas threads=none")
         conflicts("^openblas threads=pthreads")
 
+    # Add cuda support
+    with when("%nvhpc"):
+        variant(
+            "cuda",
+            default=False,
+            description="Build with CUDA, OpenMP on is strongly suggested for performance",
+        )
+        with when("+cuda"):
+            # GPUs are enabled since v6.6
+            conflicts("@:6.5")
+            # cuda version >= 10.1
+            # conflicts("cuda@:10.0.130")
+            # bugs with following nvhpcsdk versions
+            conflicts(
+                "%nvhpc@21.11:22.2", msg="QE-GPU has issues with these NVHPC compiler versions"
+            )
```

Comment:
Should be conflict with `+openmp` only

Reply:
added in 487d76f99948010ddf0daf8cd1eaf35a2dd9929e